doc/api/esm.md

```diff
@@ -726,13 +733,15 @@ console.log(M);
 executed under:
 
 ```bash
-node --experimental-wasm-modules index.mjs
+node index.mjs
 ```
 
 would provide the exports interface for the instantiation of `library.wasm`.
 
 ### Wasm Source Phase Imports
 
+> Stability: 1.2 - Release candidate
```

Comment:
It seems a little bit out of order that a release candidate feature is nested under an active development feature. I think we should make "Wasm Source Phase Imports" as an H2 section.

Reply:
Thanks for the updated review. I thought about separating source phase imports as well at first, but on further thought:

* The ESM Integration is a single specification, which is under active development, where source phase imports are part of this specification and story.
* Under that overall specification, parts of it are fully stabilized (source phase imports / string builtins)

Therefore I think it actually does make sense to have stable sub-sections of an overall unstable main section here.